src/odemis/driver/picoquant.py

```diff
@@ -681,6 +681,7 @@ def _acquire(self):
                     md = self._metadata.copy()
                     md[model.MD_ACQ_DATE] = tstart
                     md[model.MD_DWELL_TIME] = tacq
+                    md[model.MD_TIME_LIST] = numpy.linspace(0, self.pixelDuration, self._shape[-1])
```

Comment:
It should be _in place_ of the `MD_PIXEL_DUR` + `MD_TIME_OFFSET`. So, at least, you should remove these two old metadata, and use the time offset also in MD_TIME_LIST.
Moreover, you might want to update the metadata whenever one of the VA changes (so including the binning, now). It's a few extra line, but the advantage is that it's possible to see the metadata from the command line, with `odemis-cli --list-prop time-correlator`, which is handy in times of debugging.

Reply:
just pushed a change